### PR TITLE
document '--exclude_rule'

### PR DESCRIPTION
**Pull request by naftali-g**
Naming convention:
 - PRs for features that are in design should have the "proposal" label
 - PRs for features that haven't landed yet should have the "future" label
 - PRs for upcoming releases should have the "release" label
 - PRs with new documentation for existing features should have the "existing feature" label

Before requesting review:
 - Make sure there is a ticket in the DOC board in Jira
 - Make sure CI is passing
   - Spell check failure may require adding backticks around code or updating `spelling_wordlist.txt`
   - See `README.md` for information about style and markdown syntax
   - If the CI Details link gives a 404, you need to log in to readthedocs.com
 - Add link to generated documentation here
   - you can find this by following the read the docs link from the CI check
 - Ask for help in #documentation

Jira ticket: https://certora.atlassian.net/browse/CERT-733
Link to generated documentation: https://certora-certora-prover-documentation--242.com.readthedocs.build/en/242/docs/prover/cli/options.html#exclude-rule-rule-name-pattern

### CODE REVIEW

File: docs/prover/cli/options.md [hunk 1]
```diff
@@ -86,6 +89,28 @@ If we want to verify only `withdraw_succeeds`, we run
...
 `certoraRun Bank.sol --verify Bank:Bank.spec --rule withdraw_succeeds withdraw_fails`
 
+Alternatively, to verify both `withdraw_succeeds` and `withdraw_fails`, we could
+run `certoraRun Bank.sol --verify Bank:Bank.spec --rule withdraw*`
+
+(--exclude_rule)=
+### `--exclude_rule <rule_name_pattern>`
+
+**What does it do?**
+It is the opposite flag to {ref}`--rule` - use it to specify a list of rules that
```
Comment (shellygr) [inline docs/prover/cli/options.md:99]: let's say what happens if a rule is matched on both? what takes precedence?

**Review by shellygr — COMMENTED** [review]
nits

File: docs/prover/cli/options.md [hunk 2]
```diff
@@ -73,10 +73,13 @@ cause a specific rule to fail; in the process of fixing the code, updating the
 rule, and understanding counterexamples, you likely want to verify only that
 specific rule.
 
+One can either specify a specific rule name, or use pattern matching with a `*`.
+
+Note that you can specify this flag multiple times to filter in several rules or rule patterns.
```
[the remark this answers is not in the document]
Comment (naftali-g) [reply docs/prover/cli/options.md:78]: yes

File: docs/prover/cli/options.md (hunk 1 above)
Comment (naftali-g) [reply docs/prover/cli/options.md:99]: will add some text about this this

**Review by shellygr — APPROVED** [review]

**Pull request merged**